artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/proton/handler/Events.java

```diff
@@ -92,6 +97,9 @@ public static void dispatch(Event event, EventHandler handler) throws Exception
             handler.onDelivery(event.getDelivery());
             break;
          default:
+            if (PN_TRACE_FRM) {
+               logger.info("event: " + event + " not being treated");
```

Comment:
This also doesn't feel like it should be tied to having proton trace its frames to me, I'd just make this a regular trace logger.